### DISCUSSION TITLE
Allow class like Tailwind work

### DISCUSSION THREAD

**Discussion by IRediTOTO** [stylify/packages#282] [Ideas & Features]
Hello, I just found your package on GG
I would like that you allow the use of the Tailwind class name. I have person project write with Tailwind, but If change to your package I don't want refactor all of these Tw.
Let's make an example of NextJS, I don't quite understand how to install this package

**Comment by Machy8** [comment]
Hi @IRediTOTO 

**1. Installation in Next**
I have removed support for bundlers (https://github.com/stylify/packages/pull/276 - haven't finished docs yet), so in Next, do the following:

Install Stylify
```
npm i -D @stylify/bundler
yarn add -D @stylify/bundler
```

Install concurrently
```
yarn add -D concurrently
npm i concurrently
```

Create stylify config `stylify.js`:
```js
const { Bundler } = require('@stylify/bundler');

const isDev = process.argv[process.argv.length - 1] === '--w';
const bundler = new Bundler({
	watchFiles: isDev,
	// Optional
	// Compiler config info https://stylifycss.com/en/docs/stylify/compiler#configuration
	compiler: {
		// https://stylifycss.com/en/docs/stylify/compiler#variables
		variables: {},
		// https://stylifycss.com/en/docs/stylify/compiler#macros
		macros: {},
		// https://stylifycss.com/en/docs/stylify/compiler#components
		components: {},
		// ...
	}
});

// This bundles all CSS into one file
// You can configure the Bundler to bundle CSS for each page separately
// See bundler link below
bundler.bundle([
	{files: ['./path/to/**/*.tsx'], outputFile: './path/to/styles/stylify.css' },
]);
```

Then import generated stylify in your next.js `_app.js`:

```
import './path/to/styles/stylify.css.';
``` 

Last step is to update package.json:

```js
"scripts": {
	"build": "npm run stylify:build & next build",
	"dev": "concurrently 'npm run stylify:dev' 'next dev",
	"stylify:build": "node stylify.js",
	"stylify:dev": "node stylify.js --w"
}
```

**2. Tailwind Classes**
I am not planning to support Tailwind classes. 
It would mean a lot of work to follow every change they make to keep classes 1:1 + to integrate it into existing configuration.



---

Let me know if you have any trouble with the NextJS configuration.

**Reply by IRediTOTO** [reply]
Thank you, I just test it, look cool, I know there are many thing to do, wish your package grow fast.
![image](https://github.com/stylify/packages/assets/15695252/72a81b6b-286b-4f44-8984-c4f0cbcbe200)

**Reply by Machy8** [reply]
Thanks! If you encounter any problem, let me know.
